wire/msgtx.go

```diff
@@ -326,6 +413,13 @@ func (msg *MsgTx) Deserialize(r io.Reader) error {
 	return msg.BtcDecode(r, 0)
 }
 
+// DeserializeWitness decodes a transaction from r into the receiver, where the
+// encoded transaction format within r may possibly utilize the new serialization
+// format created to encode transactions bearing witness data within inputs.
+func (msg *MsgTx) DeserializeWitness(r io.Reader) error {
+	return msg.BtcDecode(r, WitnessVersion)
```

Comment:
Right, I realized you changed it as I got through the diffs and also read through the BIPs after making this comment, so I understand why the extra parameter is needed.  Feel free to ignore this.